pom.xml

```diff
@@ -945,6 +954,48 @@
                 <module>assembly</module>
             </modules>
         </profile>
+        <profile>
+            <id>spark_3.0</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>spark_3.0</value>
+                </property>
+            </activation>
+            <properties>
+                <cdh.parcel.version>6.3.0</cdh.parcel.version>
+                <envClassifier>spark_3.0</envClassifier>
+                <envHbase>hbase2.1.4</envHbase>
+                <hadoop.version>3.0.0-cdh6.3.0</hadoop.version>
+                <hbase.version>2.1.0-cdh6.3.0</hbase.version>
+                <hive.version>2.1.1-cdh6.3.0</hive.version>
+                <zookeeper.version>3.4.5-cdh6.3.0</zookeeper.version>
+                <scala.binary.version>2.12</scala.binary.version>
+                <scala.version>2.12.10</scala.version>
```

Comment:
Probably not a blocker, but is it better to use the lastest scala version [2.12.12](https://github.com/scala/scala/releases/tag/v2.12.12) or are there reasons to stay on [2.12.10](https://github.com/scala/scala/releases/tag/v2.12.10) ?

Reply:
I took 2.12.10 as spark3 has this version in the dependencies list, but should be compatible with 2.12.12. Will upgrade.

Reply:
Upgraded to 2.12.12 in the configuration